src/app/products/loan-products/create-loan-product/create-loan-product.component.ts

```diff
@@ -99,6 +129,10 @@ export class CreateLoanProductComponent implements OnInit {
     this.creditAllocation = paymentAllocation;
   }
 
+  setSupportedInterestRefundTypes(supportedInterestRefundTypes: StringEnumOptionData[]): void {
```

Comment:
Use the `OptionData` already defined in 'app/shared/models/option-data.model'

Reply:
Its not the same. OptionData id is number. StringEnumOptionData id is string